### PR TITLE
List Trust Eth reOrigin (TEO)

### PR DESCRIPTION
**Pull request by trustfarm-dev**
List Trust Eth reOrigin (TEO)
Author: trustfarm.info@gmail.com
Date: Thu April 16 14:00 GMT+9

List Trust Eth reOrigin (TEO)
Official project URL: https://forum.tao.foundation
Official block explorer URL: https://explorer.tao.foundation , https://explorer2.tao.foundation
Official git repository URL: https://github.com/tao-foundation

### CODE REVIEW

File: assets/src/main/java/bisq/asset/coins/TEO.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+package bisq.asset.coins;
+
+import bisq.asset.Coin;
+import bisq.asset.DefaultAddressValidator;
+
+public class TEO extends Coin {
+
+    public TEO() {
+        super("Trust Ether reOrigin", "TEO", new DefaultAddressValidator());
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/TEO.java]: Do not use deprecated validator

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

File: assets/src/main/java/bisq/asset/coins/TEO.java (hunk 1 above)
Comment (trustfarm-dev) [reply assets/src/main/java/bisq/asset/coins/TEO.java]: Thanks for review ? can I use EtherAddressValidator() ?, it is compatible wallet address with Ethereum.

Comment (trustfarm-dev) [reply assets/src/main/java/bisq/asset/coins/TEO.java]: Can you inform me a reference code? 
above DefaultAddressValidator() codes refer from Ethereum Classic.
thanks,

**Review by trustfarm-dev — COMMENTED** [review]
https://github.com/bisq-network/bisq/pull/2720/commits/dfccd27d74fc408975dd95524a2741c5cf621c20
Changed Address Validator to Custom Regex Ether Compatible Validater.

**Comment by trustfarm-dev** [comment]
@blabno  Can check review updated codes? thanks.

**Comment by blabno** [comment]
Code looks good, please squash the commits into a single one with a proper message.

**Comment by trustfarm-dev** [comment]
@blabno  OK I did it new PR.
https://github.com/bisq-network/bisq/pull/2775

Please review PR again, thanks, and close this PR now.

**Pull request closed without merging**